tasks/stage-buildpack/task.yml

```diff
@@ -19,6 +19,7 @@ image_resource:
   type: docker-image
   source:
     repository: pcfnorm/rootfs
+    tag: 1.0.28
```

Comment:
why are we hard-coding the rootfs version?

Reply:
Because latest of our docker image is no longer updated. Remember when we started versioning the docker image. Latest has remained an older version as to not break older pipelines.